static/src/javascripts/projects/common/modules/commercial/sticky-ad-banner.js

```diff
@@ -124,11 +124,12 @@ define([
         var transitionTimingFunction = 'cubic-bezier(0, 0, 0, .985)';
         var transitionDuration = '1s';
 
-        els.$header.css({
+        els.$page.css({
+            'will-change': 'transform',
```

Comment:
:joy: 

The strangest thing about `will-change` is that it's not supposed to appear in a stylesheet. I'm not sure one animation frame is enough to let the browser do whatever groundwork it can do, I don't think you need to be that extreme and in fact could set the property when the DOM is ready for instance.

As for removing it, why not wait for the `transitionend` event?